index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="menu-wrap">
+      <div class="logo">
+        <span class="brand">Saw Mon Han</span>
```

Comment:
- [ ] As this is our site logo, I think it would be better to add an ```<a>``` tag, and this is for three reasons: **convention, semantics, and user experience(UX)**. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. ✌✌